sonar-plugin-api/src/main/java/org/sonar/api/measures/CoreMetrics.java

```diff
@@ -36,15 +36,22 @@
 
   // the following fields are not final to avoid compile-time constants used by plugins
   public static String DOMAIN_SIZE = "Size";
+  public static String DOMAIN_COVERAGE = "Coverage";
+
+  /**
+   * @deprecated in 5.5. Merged into {@link #DOMAIN_COVERAGE}
+   */
+  @Deprecated
   public static String DOMAIN_TESTS = "Tests";
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1133)